Mini Project 1.ipynb

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
I realized that I can't leave comments on a jupyter notebook, so in the future you should do your projects in python files. It's also helpful to have something that you can run from the command line.